tutorials/erpnext/index.mdx

```diff
@@ -272,13 +290,13 @@ As the ERPNext stack's major prerequisites are installed, go on by installing th
 
 ### Installing the Frappe Framework
 
-1. During the Frappe Framework installation, you may exceed Ubuntu’s file limit, increase it, and use the following command:
+1. Increase Ubuntu’s file limit, which may be necessary for sucessfully installing the Frappe Framework. Use the following command:
 
   ```
```

Comment:
```suggestion
  `echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p`
```

Reply:
I don't think the formatting here changes anything one way or the other? :-)